UndertaleModLib/UndertaleModLib.csproj

```diff
@@ -30,11 +30,13 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="PresentationCore" />
     <Reference Include="System" />
     <Reference Include="System.Drawing" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Net.Http" />
+    <Reference Include="WindowsBase" />
```

Comment:
UndertaleModLib should avoid having any Windows-specific dependencies so that it can be easily used in cross-platform projects

Reply:
Hmm. What am I even using that for? I can't find it lol. Just a moment.

Reply:
Odd. "Project file must include the .NET Framework assembly 'WindowsBase' in the reference list."

Reply:
I'm not very familiar with C#, so I don't really know how to deal with that.

Reply:
If I were to guess, it might be the image exporter, since it uses "System.Windows". I'm not sure if that's windows like the OS, or something else GUI related though. I'll wait for your input.

Reply:
Yeah, the `System.Windows.Media` namespace is part of WPF which is not supported outside of Windows (https://www.mono-project.com/docs/gui/wpf/) and it's the reason why the main UndertaleModTool GUI is Windows-only. It may be better to use the classes from `System.Drawing` instead (and it looks like that one is even already included)

In case you find it useful when implementing this, here is the code I used when I made the Jevil hitboxes video that uses System.Drawing for drawing on the sprites: https://gist.github.com/krzys-h/81ee306d4b799972ed50c62b71d5c8cc

Reply:
Thanks. Should help me.